examples/pytorch_imagenet_resnet50.py

```diff
@@ -85,7 +84,11 @@
 verbose = 1 if hvd.rank() == 0 else 0
 
 # Horovod: write TensorBoard logs on first worker.
-log_writer = tensorboardX.SummaryWriter(args.log_dir) if hvd.rank() == 0 else None
+try:
+    import tensorboardX
```

Comment:
PyTorch 1.2 and above support Tensorboard natively, which we use in the [TorchEstimator](https://github.com/horovod/horovod/blob/master/horovod/spark/torch/remote.py#L25).  Maybe we can check that here as well?  Something like:

```
try:
    if LooseVersion(torch.__version__) >= LooseVersion('1.2.0'):
        from torch.utils.tensorboard import SummaryWriter
    else
        from tensorboardX import SummaryWriter
    log_writer = SummaryWriter(args.log_dir) if hvd.rank() == 0 else None
except ImportError as e:
    log_writer = None
```
```

Reply:
Nice, i did not know about `torch.utils.tensorboard`. @praguta And `except ImportError:` instead of `except ImportError as e:`